Drive_2_0_fieldCentric.java

```diff
@@ -237,6 +241,20 @@ public void runOpMode() {
           armraise.setPower(driverCmd_ArmRaise * MAX_ARM_LOWER_POWER);
         }
       }
+      
+      // run arm to set positions using buttons
+      // a -> ground, b -> second stacked pixel, y -> holding
+      // x grabs/ lets go of pixel (still)
+      if (DriverCmd_ArmToGround) {
+        isArmHolding = true;
+        armraise.setTargetPosition(0);
+      } else if (DriverCmd_GrabTopPixel) {
+        isArmHolding = true;
+        armraise.setTargetPosition(4.557);
+      } else if (DriverCmd_ArmToHolding) {
+        isArmHolding = false;
+        armraise.setTargetPosition(MIN_ARM_RAISE_TICKS_WHEN_NOT_PICKING_UP);
+      }
```

Comment:
I think instead of `armraise.setTargetPosition(<something>);` 
you want `armRaiseTargetPosition = <something>;`
This will then fall into / use the code from line 203-216 (on the next loop) to drive to this variable.
I'm concerned these code targets will "fight" with the code on those lines otherwise.